app/helpers/reservation_helper.rb

```diff
@@ -0,0 +1,27 @@
+module ReservationHelper
+  def text_color(status)
+    case status
+    when 'created'
+      'text-info'
+    when 'confirmed'
+      'text-success'
+    when 'cancelled'
+      'text-danger'
+    end
+  end
+
+  def update_links(reservation)
+    case reservation.status
+    when 'created'
+      content_tag(:ul, class: 'p-0 d-flex mb-3') do
+        concat(button_to('Confirm', seance_reservation_path(reservation.seance, reservation, status: 'confirmed'),
+                         method: :patch, class: 'btn btn-success me-2'))
+        concat(button_to('Cancel', seance_reservation_path(reservation.seance, reservation, status: 'cancelled'),
+                         method: :patch, class: 'btn btn-danger'))
```

Comment:
and also here. So as you can see you have a lot of different places you are using statuses and you can see how many places you would have to think of when you decide to change something